[Highly Customized] Enable Building Production Queue (#1506)
Phobos-developers/Phobos: docs/Fixed-or-Improved-Logics.md

Units.UseRepairCost=  ; boolean
```

### Enable Building Production Queue

> - Buildings can now be able to queue up to construct like other units by setting `ExpandBuildingQueue` to true. Tip: If the building is not placed, it is the same as pausing.

Reviewer: ````suggestion
- Buildings can now be queued for construction like other units if `ExpandBuildingQueue` is set to true.

```{note}
When the building becomes ready to be placed, the next building's construction will not begin until the player places the current building.
```
````

Reviewer: Also, I am not sure, maybe this should be put into "User Interface" documentation file?

Author: I think both should make sense.

In `rulesmd.ini`:
```ini
[General]

> ExpandBuildingQueue=false  ; boolean

Reviewer: I propose to name the tag simply `BuildingProductionQueue`. I think extend/expand fits only when it toggles a group of changes, like extended tooltips, here it's one specific change.

Units.UseRepairCost=  ; boolean
```

### Enable Building Production Queue

> - Buildings can now be queued for construction like other units if `ExpandBuildingQueue` is set to true.

Reviewer: ```suggestion
- Buildings can now be queued for construction like other units if `BuildingProductionQueue` is set to true.
```
